packages/dds/merge-tree/package.json

```diff
@@ -79,7 +79,7 @@
 	},
 	"devDependencies": {
 		"@fluid-internal/stochastic-test-utils": ">=2.0.0-internal.3.3.0 <2.0.0-internal.4.0.0",
-		"@fluid-tools/benchmark": "^0.45.0",
+		"@fluid-tools/benchmark": "^0.46.0",
```

Comment:
Benchmark is used by several packages in the client release group. pnpm does not _require_ all of the packages in the workspace to have matching versions like Lerna did, but we should probably update them in sync. I think there's a particular invocation of `flub` that would do it but don't know it by heart. Or a manual search-and-replace shouldn't be too hard, there's not that many instances.